.vscode/launch.template.json

```diff
@@ -40,6 +40,10 @@
       "env": {
         "NODE_ENV": "testing"
       },
+      "outFiles": [
+        "${workspaceFolder}/built/**/*.js",
```

Comment:
For future proofing maybe `{js,mjs,cjs}` though...

Reply:
```suggestion
        "${workspaceFolder}/built/**/*.{js,mjs,cjs}",
```

Reply:
Oops, this syntax doesn't actually work in this position. I guess these globs are different?